SparkPost/2web2ui
# FE-1155 - Override old Matchbox design tokens with Hibana design tokens in _critical.scss

[FE-1155](https://sparkpost.atlassian.net/browse/FE-1155)

**Note:** This branch is based on `FE-1329` and will be merged in to that branch so that it can ship with the Hibana launch

### What Changed

- Replaces OG Matchbox design tokens in `_critical.scss` with design tokens from Hibana Matchbox.

### How To Test

- Run the app locally - hit hard refresh to see the loading state that receives these styles
- Run a production build of the app as well to see the same thing and ensure it renders as expected - `npm run build && npm run serve-app`

### To Do
